pyproject.toml

```
@@ -0,0 +1,33 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {name = "Tom Hu", email = "thomas.hu@sentry.io"},
+    {name = "Codecov Support", email = "support@codecov.io"},
+]
```

Would this better belong in the “maintainers” field?
```suggestion
]
maintainers = [
    {name = "Codecov Support", email = "support@codecov.io"},
]
```